node/resolvers/checkout/index.ts

```diff
@@ -152,13 +152,53 @@ export const fieldResolvers = {
 const replaceDomain = (host: string) => (cookie: string) =>
   cookie.replace(/domain=.+?(;|$)/, `domain=${host};`)
 
+const getSession = (ctx: Context) => {
+  return sessionQueries.getSession({}, {}, ctx).catch(err => {
+    // todo: log error using colossus
+    console.error(err)
+    return {} as SessionFields
+  }) as SessionFields
+}
```

Comment:
Instead of casting you could say what is the return of the function?

```suggestion
const getSession = (ctx: Context): SessionFields => {
  return sessionQueries.getSession({}, {}, ctx).catch((err: any): SessionFields => {
    // todo: log error using colossus
    console.error(err)
    return {}
  })
}
}
```